bowtie/_report.py

```diff
@@ -1,13 +1,15 @@
 from __future__ import annotations
 
 from datetime import UTC, datetime
+from functools import reduce
 from typing import TYPE_CHECKING, TypedDict
 import importlib.metadata
 import json
 import sys
 
 from attrs import asdict, field, frozen
 from attrs.filters import exclude
+from packaging import version
```

Comment:
What's this used for? It's suspicious to use it -- this implements Python packaging versioning -- that doesn't apply across all languages necessarily.